smart-send-logistics/includes/frontend/class-ss-shipping-frontend.php

```diff
@@ -284,9 +352,20 @@ public function display_ss_shipping_agent($order)
 
                 $ordered_agent = SS_SHIPPING_WC()->get_ss_shipping_wc_order()->get_ss_shipping_order_agent($order_id);
 
+
                 $formatted_address = $this->get_formatted_address($ordered_agent, -1);
                 // Display in block instead of one line
-                $formatted_address = str_replace(',', '<br/>', $formatted_address);
+                // $formatted_address = str_replace(',', '<br/>', $formatted_address);
+
+                if (gettype($ordered_agent) == 'object') {
+                    $formatted_address = $this->get_formatted_address($ordered_agent, -1);
+                } else {
+                    $ordered_agent = explode(':', $ordered_agent);
+                    $aggentdetails = $ordered_agent[1];
+                    $formatted_address = str_replace(',', '<br/>', $aggentdetails);
+                }
```

Comment:
Why is this added, is that needed anymore?

Reply:
The purpose of above code to tackle a possible error of returning of a different type instead of object.

Reply:
And when would we return a different type?

Reply:
?

Reply:
it is removed because it was an extra precautions.